src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesComputer.java

```diff
@@ -34,6 +38,7 @@ public void taskCompleted(Executor executor, Queue.Task task, long durationMS) {
 
     @Override
     public void taskCompletedWithProblems(Executor executor, Queue.Task task, long durationMS, Throwable problems) {
+        cleanupPodAnnotation(task);
```

Comment:
Why do you remove the annotation afterwards? Isn't that information also useful after the pod has finished?

Reply:
No specific reasons I thought it was a good idea to cleanup. Is there any way some other build may be scheduled on this pod ?